infra/azure.tf

```diff
@@ -145,14 +145,31 @@ CRON
 
 chmod +x /root/daily-reset.sh
 
+cat <<GET_TARGETS > /root/get-targets.sh
+#!/usr/bin/env bash
+set -euo pipefail
+
+arg_target="$1"
+
+month=$(date +%m)
+day_of_month=$(date +%d)
+if [[ "$month" -eq 12 && "$day_of_month" -gt 22 ]]; then
+  # We treat the days after December 22nd as weekend days.
+  target='low'
+else
+  target="$arg_target"
+fi
+
+sizes='{"high": {"du1":10,"du2":0,"dw1":5,"dw2":0}, "low": {"du1":2,"du2":0,"dw1":1,"dw2":0}}'
+echo "$sizes" | jq -r --arg target "$target" '.[$target] | @json'
+GET_TARGETS
+
+chmod +x /root/get-targets.sh
+
 cat <<CRONTAB >> /etc/crontab
-30 5 * 1-11 1-5 root /root/daily-reset.sh '{"du1":10,"du2":0,"dw1":5,"dw2":0}' >> /root/log 2>&1
-30 18 * 1-11 1-5 root /root/daily-reset.sh '{"du1":2,"du2":0,"dw1":1,"dw2":0}' >> /root/log 2>&1
-30 5 * 1-11 6,7 root /root/daily-reset.sh '{"du1":2,"du2":0,"dw1":1,"dw2":0}' >> /root/log 2>&1
-30 5 1-22 12 1-5 root /root/daily-reset.sh '{"du1":10,"du2":0,"dw1":5,"dw2":0}' >> /root/log 2>&1
-30 18 1-22 12 1-5 root /root/daily-reset.sh '{"du1":2,"du2":0,"dw1":1,"dw2":0}' >> /root/log 2>&1
-30 5 1-22 12 6,7 root /root/daily-reset.sh '{"du1":2,"du2":0,"dw1":1,"dw2":0}' >> /root/log 2>&1
-30 5 23-31 12 * root /root/daily-reset.sh '{"du1":2,"du2":0,"dw1":1,"dw2":0}' >> /root/log 2>&1
+30 5 * * 1-5 root /root/daily-reset.sh \$(/root/get-targets.sh 'high') >> /root/log 2>&1
```

Comment:
at first I just changed `/root/daily-reset.sh` to take the `high|low` argument, but that wouldn't allow us to call it with  custom targets, so I added `/root/get-targets.sh` as an indirection